# Request: empty
---
# Response: string value "safe" or "unsafe"
string state

